src/test/java/org/helpberkeley/memberdata/ControlBlockRestaurantTemplateTest.java

```diff
@@ -232,4 +233,25 @@ public void noPicsRestaurantTest() {
             }
         }
     }
+
+    @Test
+    public void missingFormulaRestaurantTest() {
+        String csvData = readResourceFile("restaurant-template-v302-missing-formula.csv");
+
+        Throwable thrown = catchThrowable(() ->
+                RestaurantTemplateParser.create(csvData).restaurants());
+        assertThat(thrown).isInstanceOf(MemberDataException.class);
+        assertThat(thrown).hasMessageContaining(MessageFormat.format(
+                RestaurantTemplateParser.MISSING_FORMULA_VALUE, "51"));
```

Comment:
General AssertJ comment...If you only expect a single, specific error message you can be more precise by using hasMessage() instead of hasMessageContaining()